basic-staging/README.md

```diff
@@ -49,13 +49,12 @@ You live in your own repository. There is a file called `file.txt`.
 - `git add`
 - `git commit`
 - `git commit -m "My lazy short commit message"`
-- `git reset`
 - `git log`
 - `git log -n 5`
 - `git log --oneline`
 - `git log --oneline --graph`
-- `git reset HEAD`
-- `git checkout`
+- `git restore --source`
```

Comment:
I would leave out this line, as --source is not relevant for this exercise and not suitable in scope for people running "basic-staging"

Reply:
I just added it since git reset HEAD was mentioned and --source is the restore equivalent. But I agree, they dont need it for this exercise.